content/agent/faq/supported-os-versions.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Datadog Agent supported OS versions
+kind: faq
+disable_toc: true
+further_reading:
+- link: "/agent/"
+  tag: "Documentation"
+  text: Learn more about the Datadog Agent
+---
+
+### Agent 5+
+
+|OS| Supported versions|
+|:----|:----|
+|Debian x86_64 | Debian 7 (wheezy) and above (we do not support SysVinit)|
+|Ubuntu x86_64 | Ubuntu 12.04 and above|
+|RedHat/CentOS x86_64| RedHat/CentOS 6 and above |
+|SUSE Enterprise Linux x86_64 | SUSE 11 SP4 and above (we do not support SysVinit)| 
+|Fedora| Fedora 25 and above |
+|MacOS| OSX 10.10 and above|
+|Windows server 64-bit| Windows server 2008r2 or above|
+|Windows 64-bit| Windows 7 or above|
+
+### Agent 6+ (Beta)
+
+|OS| Supported versions|
+|:----|:----|
+|Debian x86_64 | Debian 7 (wheezy) and above (we do not support SysVinit)|
+|Ubuntu x86_64 | Ubuntu 12.04 and above|
```

Comment:
`14.04`

Reply:
This definitely makes sense for agent 6.

Reply:
👍